transports/core-transports-tests/src/test/java/org/mule/compatibility/core/endpoint/outbound/OutboundResponsePropertiesMessageProcessorTestCase.java

```diff
@@ -1,60 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.compatibility.core.endpoint.outbound;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
-import static org.mule.runtime.core.MessageExchangePattern.REQUEST_RESPONSE;
-
-import org.mule.compatibility.core.api.endpoint.EndpointBuilder;
-import org.mule.compatibility.core.api.endpoint.OutboundEndpoint;
-import org.mule.compatibility.core.endpoint.AbstractEndpointBuilder;
-import org.mule.compatibility.core.processor.AbstractMessageProcessorTestCase;
-import org.mule.runtime.core.api.Event;
-import org.mule.runtime.core.api.message.InternalMessage;
-import org.mule.runtime.core.api.processor.InterceptingMessageProcessor;
-import org.mule.runtime.core.session.DefaultMuleSession;
-
-import org.junit.Test;
-
-public class OutboundResponsePropertiesMessageProcessorTestCase extends AbstractMessageProcessorTestCase {
```

Comment:
Because it depends on behaviour of AbstractRequestResponseMP which no longer exists.